src/test/core.ts

```diff
@@ -311,7 +322,8 @@ describe('snabbdom', function() {
         prevElm.id = 'id';
         prevElm.className = 'class';
         var text = new Text('Foobar');
-        text.testProperty = function () {}; // ensures we dont recreate the Text Node
+        const reference = {};
+        (text as any).testProperty = reference; // ensures we dont recreate the Text Node
```

Comment:
Is there a reason this is specifically a function and not any other reference? In TypeScript, our ESLint does complain about empty functions. So I changed this to an empty object. Cool?